src/content/documentation/publish.md

```diff
@@ -30,7 +30,7 @@ Learn how to get your extension signed and distributed worldwide or to your ente
 
 ## Get your extension signed
 
-Extensions and themes need to be [submitted for signing](/documentation/publish/signing-and-distribution-overview/) by Mozilla before they can be installed in release and beta versions of Firefox for use in the [release](https://www.mozilla.org/firefox/) and [Beta](https://www.mozilla.org/firefox/channel/desktop/) versions of Firefox. Signing provides Firefox users with the assurance that an extension hasn’t been tampered with and gives Mozilla the ability to block malicious extensions. 
+Extensions and themes need to be [submitted for signing](/documentation/publish/signing-and-distribution-overview/) by Mozilla before they can be installed and used in the [Release](https://www.mozilla.org/firefox/) and [Beta](https://www.mozilla.org/firefox/channel/desktop/) versions of Firefox. Signing provides Firefox users with the assurance that an extension hasn’t been tampered with and gives Mozilla the ability to block malicious extensions. 
```

Comment:
"release" is just a description, not a proper name, so it should remain in lowercase.
```suggestion
Extensions and themes need to be [submitted for signing](/documentation/publish/signing-and-distribution-overview/) by Mozilla before they can be installed and used in the [release](https://www.mozilla.org/firefox/) and [Beta](https://www.mozilla.org/firefox/channel/desktop/) versions of Firefox. Signing provides Firefox users with the assurance that an extension hasn’t been tampered with and gives Mozilla the ability to block malicious extensions. 
```